### PR TITLE
Draft:1117-support-button-placement

### PR DESCRIPTION
**Pull request by rajkarnikar11**

### CODE REVIEW

**Comment by man-git007** [comment]
@rajkarnikar11 What is the status of this PR?

cc @SauravSilwal10

File: resources/assets/js/app.ts [hunk 1]
```diff
@@ -342,6 +342,25 @@ const stickySidebar = (
     }
   }
 };
+window.onload = () => {
+  setTimeout(() => {
+    const supportButton: HTMLElement = document.querySelector(
+      '#launcher'
+    ) as HTMLElement;
+    if (supportButton !== null) {
```
Comment (sriza) [inline resources/assets/js/app.ts:350]: new line

File: resources/assets/js/views/activity/bulk-publish/BulkPublishing.vue [hunk 2]
```diff
@@ -110,22 +110,39 @@ let intervalID;
  *   Component lifecycle - onMounted
  */
 onMounted(() => {
+  console.log('mounted');
```
Comment (sriza) [inline resources/assets/js/views/activity/bulk-publish/BulkPublishing.vue:113]: remove console log

File: resources/assets/js/components/sections/ProgressLoader.vue [hunk 3]
```diff
@@ -32,10 +32,22 @@ defineProps({
 
 onMounted(() => {
   document.body.classList.add('overflow-y-hidden');
+  const supportButton: HTMLElement = document.querySelector(
+    '#launcher'
+  ) as HTMLElement;
+  if (supportButton !== null) {
```
Comment (sriza) [inline resources/assets/js/components/sections/ProgressLoader.vue:38]: add a new line

File: resources/assets/js/components/sections/ProgressLoader.vue [hunk 4]
```diff
@@ -32,10 +32,22 @@ defineProps({
...
+    supportButton.style.display = 'none';
+  }
 });
 
 onUnmounted(() => {
   document.body.classList.remove('overflow-y-hidden');
+  const supportButton: HTMLElement = document.querySelector(
+    '#launcher'
+  ) as HTMLElement;
+  if (supportButton !== null) {
```
Comment (sriza) [inline resources/assets/js/components/sections/ProgressLoader.vue:48]: add a new line

**Review by sriza — CHANGES_REQUESTED** [review]

**Review by sriza — APPROVED** [review]

**Pull request merged**